inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/workflow/group/DeleteGroupWorkflowDefinition.java

```diff
@@ -85,22 +77,12 @@ public WorkflowProcess defineProcess() {
         deleteMQTask.setListenerFactory(groupTaskListenerFactory);
         process.addTask(deleteMQTask);
 
-        // Delete Sort
```

Comment:
When do we need to delete the Sort task?

As far as I know, a Sort task may process data of multiple Pairs<StreamSource, StreamSink>. To modify any StreamSource or StreamSink, we stop the Sort task first, push all updated and non-updated StreamSource or StreamSink information to the Sort task, and finally restart it.

In this case, how to maintain the Sort task?

Reply:
The current sort configuration belongs to the group. The sort configuration will not be performed until the stream configuration under the current group is successful.